app/src/main/java/edu/uc/zhukv/droneradarmap/MapsActivity.kt

```diff
@@ -34,12 +36,27 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
 //        mapFragment.getMapAsync(this)
 
         spinner = findViewById(R.id.tileType)
-        val tileName = arrayOf("Clouds", "Temperature", "Precipitations", "Snow", "Rain", "Wind", "Sea level press.")
-        val adpt: ArrayAdapter<String> = ArrayAdapter(this, android.R.layout.simple_spinner_item, tileName)
-        spinner?.adapter = adpt
+        val tileName = arrayOf(
+            "Clouds",
```

Comment:
Updated the formatting of the array in accords to the Kotlin Style Guide. See: https://developer.android.com/kotlin/style-guide